sale_stock_picking_blocking/tests/test_sale_stock_picking_blocking.py

```diff
@@ -2,10 +2,10 @@
 #   (http://www.forgeflow.com)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 from odoo.exceptions import ValidationError
-from odoo.tests import Form, common
+from odoo.tests import Form, TransactionCase
```

Comment:
While you are at it, inherit from `odoo.addons.base.tests.common.BaseCommon` to circumvent tracking for better performance.